layouts/default.vue

```diff
@@ -29,55 +40,25 @@ export default {
   },
   methods: {
     getLocalStorage() {
-      const loggedSessionCache = sessionStorage.getItem("logged");
-      if (loggedSessionCache !== null && loggedSessionCache === "true") {
-        this.logged = true;
-      } else {
-        const loggedBrowserCache = localStorage.getItem("logged");
-
-        this.logged = loggedBrowserCache !== null && loggedBrowserCache === "true";
-        sessionStorage.setItem("logged", this.logged);
-      }
-
-      const usernameSessionCache = sessionStorage.getItem("username");
-      if (usernameSessionCache !== null && usernameSessionCache !== "undefined") {
-        this.username = usernameSessionCache;
-      } else {
-        const usernameBrowserCache = localStorage.getItem("username");
-
-        this.username = usernameBrowserCache !== null && usernameBrowserCache !== "undefined"
-          ? usernameBrowserCache
-          : '';
-        sessionStorage.setItem("username", this.username);
+      this.logged = localStorage.getItem("logged") === "true";
+      if (this.logged === null) {
+        this.logged = false;
       }
 
-      const emailSessionCache = sessionStorage.getItem("email");
-      if (emailSessionCache !== null && emailSessionCache !== "undefined") {
-        this.email = emailSessionCache;
-      } else {
-        const emailBrowserCache = localStorage.getItem("email");
-
-        this.email = emailBrowserCache !== null && emailBrowserCache !== "undefined"
-          ? emailBrowserCache
-          : '';
-        sessionStorage.setItem("email", this.email);
+      this.username = localStorage.getItem("username");
+      if (this.username === null) {
+        this.username = "";
       }
 
-      const pictureSessionCache = sessionStorage.getItem("profilePicture");
-      if (pictureSessionCache !== null && pictureSessionCache !== "undefined") {
-        this.profilePicture = pictureSessionCache;
```

Comment:
Questa parte e fino in fondo rimettila come era prima, altrimenti non va il ricordami